stampery-trailbot-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,211 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Username for the Virtual Machine."
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for the Virtual Machine."
+      }
+    },
+    "adminEmail": {
+      "type": "string",
+      "metadata": {
+        "description": "Your email address."
+      }
+    },
+    "dnsLabelPrefix": {
+       "type": "string",
+       "metadata": {
+          "description": "Hostname assigned to the Public IP, also used as the VM Name. Must be lowercase. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+       }
+    },
+    "clientPubKey": {
+      "type": "string",
```

Comment:
Good observation. The thing is that the client public key is not secret itself.
In addition, it is a PGP armored public key that you need to paste into the corresponding field when using this quickstart template. If the field was a secureString and you pasted the armor, it would be shown as a row of black dots, giving a quite bad UX when compared to showing the actual content of the armor confirming it has been pasted properly.